hnn_core/opt_toy_example/metrics.py

```diff
@@ -0,0 +1,74 @@
+"""Metrics for parameter optimization."""
+
+# Authors: Carolina Fernandez <cxf418@miami.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import numpy as np
+
+from hnn_core import simulate_dipole, MPIBackend
+
+from scipy.signal import resample
+
+
+def _rmse_evoked(net, initial_params, set_params, predicted_params,
+                 update_params, obj_values, scale_factor, smooth_window_len,
+                 tstop, target):
```

Comment:
can you refactor this with 

https://github.com/jonescompneurolab/hnn-core/blob/c81e5b87fbcc13bd4c6e5a403e838ca63ea16760/hnn_core/dipole.py#L239

maybe move the old function here and update it?